Rolo123y br1

Rolo123y opened the pull request:
restructure the app to include:
- routes in routes.py
- database tables in models.py
- html pages in templates

to run the project: 
- open cmd/shell and cd to fydb_bess
- type and enter: python run.py

Rolo123y committed:
create branch rolo123y-br

Rolo123y committed:
rstructure project

gaurav613 commented on app/routes.py:
```diff
@@ -0,0 +1,17 @@
+
+@app.route("/Profile")
+def Profile_page():
+    return render_template('Profile.html')
```
Looks like `Profile.html` doesn't exist in the templates, which causes a TemplateNotFound error when opened. Has a template been created? If not, a blank page can be created for now, just so there are no errors in the initial setup.

Rolo123y replied:
added empty profile page which can be accessed with either:
/profile
/profile/someusername

gaurav613 commented on app/templates/home.html:
```diff
@@ -0,0 +1,69 @@
+        <tbody>
+            {% for item in items %}
+            <tr>
+                <td>{{item.MonthYear}}</td>
```
Month column is showing empty, not sure if it's a db error or just the way this is being called.

Rolo123y replied:
fixed

gaurav613 commented on app/templates/home.html:
```diff
@@ -0,0 +1,69 @@
+    <p>Welcome to Batteryistic, with Battery Energy Storage Systems, you can start saving and thriving</p>
+</div>
+<div class="container">
+    <button class="btn btn-outline btn-success">Start Saving</button>
```
This is more of a future improvement, but maybe clicking this button can be used to display the cost-saving table below, instead of showing it from the start.

gaurav613 reviewed (changes requested):
Left a few comments - regarding the profile page, savings button, and table.

Rolo123y committed:
- Added empty Profile page
- clicking the 'start saving' btn will display a table
- Month/Year column in table now displays correctly

gaurav613 reviewed (approved):
Looks good now - the project is structured well and performs basic functionality.
